src/main/java/uk/gov/hmcts/reform/sscs/helper/HearingsMapping.java

```diff
@@ -40,6 +39,13 @@ public static HearingRequestPayload buildHearingPayload(HearingWrapper wrapper)
         return requestPayloadBuilder.build();
     }
 
+    public static HearingRequestPayload buildUpdateHearingPayload(HearingWrapper wrapper) {
+        HearingRequestPayload payload = buildHearingPayload(wrapper);
+        Long versionNumber = wrapper.getOriginalCaseData().getSchedulingAndListingFields().getActiveHearingVersionNumber();
+        payload.getRequestDetails().setVersionNumber(versionNumber);
```

Comment:
Change to payload.getRequestDetails().setVersionNumber(HearingsRequestMapping.getVersion(wrapper.getUpdatedCaseData()));